solidity/test/isms/MOfNTestUtils.sol

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity >=0.8.0;
+
+library MOfNTestUtils {
+    function choose(
+        uint8 m,
+        uint256[] memory choices,
+        bytes32 seed
+    ) internal pure returns (uint256[] memory) {
+        uint256 bitmask = _bitmask(m, uint8(choices.length), seed);
```

Comment:
https://github.com/jtriley-eth/ez-bitmap for future reference

Reply:
This would have saved me some time